WordPressAuthenticatorTests/SupportingFiles/WordPressAuthenticatorTests-Bridging-Header.h

```diff
@@ -0,0 +1,3 @@
+//
+//  Use this file to import your target's public headers that you would like to expose to Swift.
+//
```

Comment:
It isn't great that I had to add this empty file, but without this file I was unable to access Swift code from ObjC.  This took me ages to figure out.